src/components/RenderReviewInformation.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from "react"
+import { patchVotesByID } from "../api";
+import { useParams } from "react-router-dom";
+
+function RenderReviewInformation({review}) {
+
+    const { id } = useParams(); 
+
+    const [votesChange, setVotesChange] = useState(0);   
+    const [error, setError] = useState("") 
+
+    const errMsg = "Your vote cannot be registered at this moment, please try again later, whilst we are trying to resolve this issue"
```

Comment:
Whenever the votes are below 0, this error is shown when a review is up- or downvoted, even though the vote still registers. It seems like there is a bug!